app/src/main/java/org/fossasia/openevent/app/data/models/Event.java

```diff
@@ -99,7 +99,7 @@
     private String organizerName;
 
     // Tells if the event saved is complete ( with tickets )
-    private boolean isComplete = true;
+    private boolean isComplete;
```

Comment:
You are setting it to false while downloading events for the first time. I am saying to let the default value be `false` in Event.class` itself. And it can be set to true at appropriate place.